packages/model-viewer/src/three-components/SmoothControls.ts

```diff
@@ -370,10 +370,10 @@ export class SmoothControls extends EventDispatcher {
 
     const deltaRatio = deltaZoom === 0 ?
         0 :
-        deltaZoom > 0 ? (maximumRadius! - radius) /
-                (Math.log(maximumFieldOfView!) - this.goalLogFov) :
-                        (radius - minimumRadius!) /
-                (this.goalLogFov - Math.log(minimumFieldOfView!));
+        ((deltaZoom > 0 ? maximumRadius! : minimumRadius!) - radius) /
+            (Math.log(
+                 deltaZoom > 0 ? maximumFieldOfView! : minimumFieldOfView!) -
+             this.goalLogFov);
```

Comment:
This isn't actually a change: the line wrapping was getting very confused and it was too long anyway, so I tried to simplify it a bit. Still not great.

Reply:
Actually the parentheses help a lot, still takes a few cycles to process though.